README.md

```diff
@@ -115,6 +115,33 @@ Encoding the payload can be done with the [Ruby `jwt` gem](https://github.com/jw
 JWT.encode payload, SECRET, 'HS256'
 ```
 
+## Settings vs config vs Environment variables
+
+Refer to the [the config gem](https://github.com/railsconfig/config#accessing-the-settings-object) to understand the `file based settings` loading order.
+
+To override file based via `Machine based env variables settings`
+```bash
+cat config/settings.yml
+file
+  based
+    settings
+      env1: 'some file based value'
+```
+
+```bash
+export SETTINGS__FILE__BASED__SETTINGS_ENV1="machine wins"
+```
+
+```ruby
+puts Settings.file.based.setting.env1
+
+# machine wins
+```
```

Comment:
Sorry to be picky, but if you're fixing the underscores above then also remove the `# ` since that wouldn't be printed out.